go.mod

```diff
@@ -12,9 +12,9 @@ require (
 	github.com/containernetworking/cni v1.1.2
 	github.com/containernetworking/plugins v1.2.0
 	github.com/containers/buildah v1.29.1-0.20230201192322-e56eb25575c7
-	github.com/containers/common v0.51.0
+	github.com/containers/common v0.51.1-0.20230220152405-6f7e45b137bd
 	github.com/containers/conmon v2.0.20+incompatible
-	github.com/containers/image/v5 v5.24.1-0.20230214095352-ae0edb7a4261
+	github.com/containers/image/v5 v5.24.1
```

Comment:
is this a downgrade? I ask because github.com/vbauerster/mpb/v7 is added back in.